x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/SqlQueryRequest.java

```diff
@@ -140,6 +140,14 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         if (query != null) {
             builder.field("query", query);
         }
+        if (requestInfo() != null) {
```

Comment:
code is repeated here: https://github.com/elastic/elasticsearch/pull/36149/files#diff-7e3b87bb842194e7f25b2a712cb821e6R46 please extract to a method.